support/src/indirect_intrusive_heap.h

```diff
@@ -196,10 +198,10 @@ namespace crimson {
     IndIntruHeap() :
       count(0)
     {
-      // empty
+      assert(K > 0);
```

Comment:
yes, there is a  compile-time assertion in the CMakeFiles, so we can get rid of it.

And yes, K = 1 is terrible of course, but it is technically a valid value for K.